features/step_definitions/artifacts.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const Assert = require('chai').assert;
+const { Before, Then, When } = require('cucumber');
+const request = require('screwdriver-request');
+
+const TIMEOUT = 240 * 1000;
+
+Before('@artifacts', function hook() {
+    this.repoName = 'functional-artifacts';
+
+    // Reset shared information
+    this.buildId = null;
+    this.pipelineId = null;
+    this.eventId = null;
+    this.jwt = null;
+});
+
+When(/^the "zipped" job is started$/, { timeout: TIMEOUT }, function step() {
+    return request({
+        url: `${this.instance}/${this.namespace}/events`,
+        method: 'POST',
+        json: {
+            pipelineId: this.pipelineId,
+            startFrom: 'zipped'
+        },
+        context: {
+            token: this.jwt
+        }
+    })
+        .then((resp) => {
+            Assert.equal(resp.statusCode, 201);
+            this.eventId = resp.body.id;
+        })
+        .then(() =>
+            request({
+                url: `${this.instance}/${this.namespace}/events/${this.eventId}/builds`,
+                method: 'GET',
+                context: {
+                    token: this.jwt
+                }
+            })
+        )
+        .then((resp) => {
+            Assert.equal(resp.statusCode, 200);
+            this.buildId = resp.body[0].id;
+        });
+});
+
+Then(/^the "zipped" build succeeds$/, { timeout: TIMEOUT }, function step() {
+    return this.waitForBuild(this.buildId).then((resp) => {
+        Assert.equal(resp.body.status, 'SUCCESS');
+        Assert.equal(resp.statusCode, 200);
+    });
+});
+
+Then(/^artifacts were found in the build$/, { timeout: TIMEOUT }, function step() {
+    const artifactName1 = 'sample1.txt';
+    const artifactName2 = 'sample2.txt';
+
+    const retryConfig = {
+        limit: 6,
+        statusCodes: [408, 404, 413, 429, 500, 502, 503, 504, 521, 522, 524],
+        calculateDelay: ({ computedValue }) => {
```

Comment:
Is it better to increase the delay time based on the number of attempts?
If the request fails, you should always wait 30 seconds.

Reply:
I was mistaken about the setting method.
It is now fixed to retry at 30-second intervals.

Reply:
Fixed the delay time to increase with the number of retries.
The maximum delay time is 30 seconds.